.github/workflows/adelphi-python.yaml

```diff
@@ -0,0 +1,29 @@
+name: adelphi Python
+
+on:
+  push:
+    branches: [ 115-breadth-first-testing ]
+  pull_request:
+    branches: [ 115-breadth-first-testing ]
```

Comment:
A temporary config (since this is branched off of the feature branch changing the integration test framework).  This will need to be changed to the default branch before we actually merge it in.